src/app/static/src/store/mutations.ts

```diff
@@ -2,9 +2,6 @@ import { RootState } from "@/store/state";
 import { ApiInfo, Metadata, Data } from "@/types";
 
 export const mutations = {
-    increment(state: RootState, amount: number): void {
```

Comment:
[This](https://github.com/mrc-ide/comet/pull/9/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.vue&file-filters%5B%5D=No+extension&hide-deleted-files=true#diff-639b6c31171c4f050decc908055911c50e51c2e13487d4a3ce5526fc404f503cR6-R10) can go too... by which time mrc-2372 is probably redundant!